builtin/settingtypes.txt

```diff
@@ -126,9 +126,9 @@ touchscreen_threshold (Touch screen threshold) int 20 0 100
 #    If disabled, virtual joystick will center to first-touch's position.
 fixed_virtual_joystick (Fixed virtual joystick) bool false
 
-#    (Android) Use virtual joystick to trigger "aux" button.
-#    If enabled, virtual joystick will also tap "aux" button when out of main circle.
-virtual_joystick_triggers_aux (Virtual joystick triggers aux button) bool false
+#    (Android) Use virtual joystick to trigger 'AUX' button.
+#    If enabled, virtual joystick will also tap 'AUX' button when out of main circle.
+virtual_joystick_triggers_aux (Virtual joystick triggers 'AUX' button) bool false
```

Comment:
Although 'AUX' is correct for the general technical term, in MT we refer to this key as 'aux', so 'aux' is better.

Reply:
See https://github.com/minetest/minetest/search?p=2&q=aux&unscoped_q=aux

Reply:
'aux' please as explained.